mrp_order_report_product_barcode/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
-    "name": "Product Barcode on the Production Order",
+    "name": "Lot Barcode on the Production Order",
```

Comment:
Why changing this? It's a product label AFAIK.

Reply:
`hooks.py` won't do anything in this case. For migration, you have to do it on OpenUpgrade part, and if you install this module in a clean 16, it doesn't matter how it was called in previous versions. The README should clarify about the scope of the module.

Reply:
Not really. I was clarifying that renaming the module has nothing to do with `hooks.py`. I think you should rename the module to reflect that is only "the lot part".

Reply:
@pedrobaeza Just for my knowledge and clarification, but a migration PR don't should be only a migration of the existing and in case of changed behaviour (or name like this) you have to do a new PR? Thanks

Reply:
Well, not really, because you can't migrate the module as it was, so for performing the migration, you need to do these transformations.

Reply:
For doing some squash:

https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

For renaming a module with its whole history:

```
git filter-branch --tree-filter 'if [ -d old_module_name ]; then mv old_module_name new_module_name; fi' HEAD
git rebase origin/branch
 ```